### PR TITLE
atlas/schema: support `cloud` config block

### PR DESCRIPTION
**Pull request by giautm**
- Support `cloud` config on `AtlasSchema` to supply ATLAS_TOKEN per resource

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -51,7 +51,6 @@ WORKDIR /
 COPY --from=builder /workspace/manager .
 COPY --from=atlas /usr/local/bin/atlas .
 RUN chmod +x /atlas
-ENV ATLAS_NO_UPDATE_NOTIFIER=1
```
Comment (giautm) [inline Dockerfile:54]: This already handle by the SDK

File: charts/atlas-operator/templates/crds/crd.yaml [hunk 2]
```diff
@@ -442,6 +442,40 @@ spec:
           spec:
             description: AtlasSchemaSpec defines the desired state of AtlasSchema
             properties:
+              cloud:
+                description: Cloud defines the Atlas Cloud configuration.
+                properties:
+                  project:
+                    type: string
+                  tokenFrom:
```
Comment (rotemtam) [inline charts/atlas-operator/templates/crds/crd.yaml:450]: Use "repo"

Comment (rotemtam) [reply charts/atlas-operator/templates/crds/crd.yaml:450]: I don't think "project" is actually used or necessary in AtlasMigration.

Comment (giautm) [reply charts/atlas-operator/templates/crds/crd.yaml:450]: Yep, I'll do rename

**Review by luantranminh — APPROVED** [review]

**Review by rotemtam — APPROVED** [review]

**Pull request merged**